### PR TITLE
DMP-3705 - only get audio that isCurrent

### PR DESCRIPTION
**Pull request by cmagowan-hmcts**
https://tools.hmcts.net/jira/browse/DMP-3705

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 public interface AudioService {
 
-    List<MediaEntity> getAudioMetadata(Integer hearingId, Integer channel);
+    List<MediaEntity> getMediaEntntitiesByHearingAndChannel(Integer hearingId, Integer channel);
```
Comment (karen-hedges) [inline src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java:12]: You have a typo here

File: src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java [hunk 2]
```diff
@@ -56,8 +56,8 @@ private AudioFileInfo createAudioFileInfo(MediaEntity mediaEntity, Path download
     }
 
     @Override
-    public List<MediaEntity> getAudioMetadata(Integer hearingId, Integer channel) {
-        return mediaRepository.findAllByHearingIdAndChannel(hearingId, channel);
+    public List<MediaEntity> getMediaEntntitiesByHearingAndChannel(Integer hearingId, Integer channel) {
```
Comment (karen-hedges) [inline src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java:59]: Same typo

**Review by tonymort — APPROVED** [review]
Minor comment

File: src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java [hunk 3]
```diff
@@ -77,7 +77,7 @@ public class AudioController implements AudioApi {
         securityRoles = {JUDICIARY, REQUESTER, APPROVER, TRANSCRIBER, TRANSLATION_QA},
         globalAccessSecurityRoles = {JUDICIARY, SUPER_ADMIN, SUPER_USER, RCJ_APPEALS, TRANSLATION_QA, DARTS})
     public ResponseEntity<List<AudioMetadata>> getAudioMetadata(Integer hearingId) {
-        List<MediaEntity> mediaEntities = audioService.getAudioMetadata(hearingId, 1);
+        List<MediaEntity> mediaEntities = audioService.getMediaEntntitiesByHearingAndChannel(hearingId, 1);
```
[the remark this answers is not in the document]
Comment (cmagowan-hmcts) [reply src/main/java/uk/gov/hmcts/darts/audio/controller/AudioController.java]: thanks, now fixed.

File: src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java (hunk 1 above)
Comment (cmagowan-hmcts) [reply src/main/java/uk/gov/hmcts/darts/audio/service/AudioService.java]: thanks, now fixed.

File: src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java (hunk 2 above)
Comment (cmagowan-hmcts) [reply src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioServiceImpl.java]: thanks, now fixed.

**Pull request merged**